cumulusci/cli/cci.py

```diff
@@ -1104,7 +1116,7 @@ def task_run(config, task_name, org, o, debug, debug_before, debug_after, no_pro
 
             pdb.set_trace()
 
-    except CumulusCIUsageError as e:
+    except (CumulusCIUsageError, TaskNotFoundError) as e:
```

Comment:
TaskNotFoundError is a subclass of CumulusCIUsageError; you don't have to catch it explicitly.